torchvision/csrc/cuda/DeformConv_cuda.cu

```diff
@@ -70,7 +70,7 @@
 #include <ATen/TensorUtils.h>
 #include <ATen/cuda/CUDAContext.h>
 #include <c10/cuda/CUDAGuard.h>
-#include <ATen/cuda/CUDAApplyUtils.cuh>
+#include <THC/THCAtomics.cuh>
```

Comment:
Is this a good idea to rely on THC? Don't we have a `ATen` or C10 equivalent?

Reply:
AFAIK I didn't see any such thing in c10 or ATen. And this file is included in `ATen/cuda/CUDAApplyUtils.cuh`, I think is something that should be migrated to ATen but not done yet.